README.md

```diff
@@ -331,6 +331,7 @@ We are trying to update our dependencies to run on `arm32v7` and `x64` boards. H
 | pihole/pihole | v4.4 | [✔️](https://raw.githubusercontent.com/pi-hole/docker-pi-hole/v4.4/Dockerfile_amd64) | [✔️](https://raw.githubusercontent.com/pi-hole/docker-pi-hole/v4.4/Dockerfile_armhf) | [✔️](https://raw.githubusercontent.com/pi-hole/docker-pi-hole/v4.4/Dockerfile_arm64) | [Github](https://github.com/pi-hole/docker-pi-hole) - [DockerHub](https://hub.docker.com/r/pihole/pihole) |
 | btcpayserver/tor | 0.4.1.5 | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Tor/0.4.1.5/Tor/0.4.1.5/linuxamd64.Dockerfile) | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Tor/0.4.1.5/Tor/0.4.1.5/linuxarm32v7.Dockerfile) | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Tor/0.4.1.5/Tor/0.4.1.5/linuxarm64v8.Dockerfile) | [Github](https://github.com/btcpayserver/dockerfile-deps) - [DockerHub](https://hub.docker.com/r/btcpayserver/tor) |
 | postgres | 9.6.5 | [✔️](https://raw.githubusercontent.com/docker-library/postgres/b7cb3c6eacea93be2259381033be3cc435649369/9.6/Dockerfile) | [✔️](https://raw.githubusercontent.com/docker-library/postgres/b7cb3c6eacea93be2259381033be3cc435649369/9.6/Dockerfile) | [✔️](https://raw.githubusercontent.com/docker-library/postgres/b7cb3c6eacea93be2259381033be3cc435649369/9.6/Dockerfile) | [Github](https://github.com/docker-library/postgres) - [DockerHub](https://hub.docker.com/_/postgres) |
+| apotdevin/thunderhub | 0.3.5 | [✔️](https://raw.githubusercontent.com/apotdevin/thunderhub/master/Dockerfile) | ️❌ | ️❌ | [Github](https://github.com/apotdevin/thunderhub) - [DockerHub](https://hub.docker.com/r/apotdevin/thunderhub) |
```

Comment:
Instead of adding this manually, can you also add the relevant build details to `contrib\DockerFileBuildHelper\Program.cs`
This file basically tells btcpayserver-docker how to automatically self build your docker image in case the user does not want to pull from DockerHub.

I think this would do
```cs
                case "bitcoin_thub":
                    dockerInfo.DockerFilePath = $"Dockerfile";
                    dockerInfo.GitLink = "https://github.com/apotdevin/thunderhub";
                    dockerInfo.GitRef = $"v{image.Tag.Split('-')[0]}";
                    dockerInfo.SupportedByUs = false;
                    break;
```

after adding that, run `update-repo.sh`

Reply:
Will do, didn't know it was generated :D

Reply:
Do I also need to add into `build-all-images.sh`? `update-repo-sh` gives me `Missing image info`